### PR TITLE
Add support for RTT cli

### PR DESCRIPTION
**Pull request by CodingRays**
Adds support for https://github.com/openthread/openthread/pull/9148

### CODE REVIEW

File: src/src/system.c [hunk 1]
```diff
@@ -139,6 +141,9 @@ bool otSysPseudoResetWasRequested(void)
 
 void otSysProcessDrivers(otInstance *aInstance)
 {
+#if OPENTHREAD_UART_RTT_ENABLE
+    utilsUartRttProcess();
+#endif
```
Comment (LuDuda) [inline src/src/system.c:146]: Could we move it inside nrf5TransportProcess? I believe motivation here is to match e.g. UART functionality, right?

**Review by LuDuda — CHANGES_REQUESTED** [review]
Looks good, one change requested.

File: src/src/system.c (hunk 1 above)
Comment (CodingRays) [reply src/src/system.c]: Yes that was the intention

File: src/src/system.c [hunk 2]
```diff
@@ -46,6 +46,8 @@
 
 #include <openthread/config.h>
 
+#include <utils/uart_rtt.h>
```
Comment (LuDuda) [inline src/src/system.c:49]: nit: do we need it?

**Review by LuDuda — APPROVED** [review]
Thanks!

**Review by jwhui — APPROVED** [review]

**Pull request merged**